com.sap.cloud.lm.sl.cf.process/src/main/java/com/sap/cloud/lm/sl/cf/process/steps/RenameApplicationsStep.java

```diff
@@ -72,15 +72,14 @@ public void execute(ExecutionWrapper execution) {
             }
 
             getStepLogger().debug(Messages.UPDATING_APP_NAMES_WITH_NEW_SUFFIX);
-            updateApplicationNamesInDescriptor(context, BlueGreenApplicationNameSuffix.NEW.asSuffix());
+            updateApplicationNamesInDescriptor(context, BlueGreenApplicationNameSuffix.IDLE.asSuffix());
         }
 
         private void renameOldApps(List<String> appsToRename, CloudControllerClient client) {
-            getStepLogger().info(Messages.RENAMING_OLD_APPLICATIONS);
-
             for (String appName : appsToRename) {
-                getStepLogger().debug(Messages.RENAMING_APPLICATION, appName);
-                client.rename(appName, appName + BlueGreenApplicationNameSuffix.OLD.asSuffix());
+                getStepLogger().info(Messages.RENAMING_APPLICATION_0_TO_1, appName,
+                                     appName + BlueGreenApplicationNameSuffix.LIVE.asSuffix());
```

Comment:
Extract `appName + BlueGreenApplicationNameSuffix.LIVE.asSuffix()` in a variable.